modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/extensions/extensionservices/ServiceExtensionMockServer.java

```diff
@@ -16,7 +16,7 @@
  * under the License.
  */
 
-package org.wso2.identity.integration.test.actions.mockserver;
+package org.wso2.identity.integration.test.extensions.extensionservices;
```

Comment:
service-extensions.mockservices

Reply:
As hyphens are not allowed shall we go with `service.extensions.mockservices` or `serviceextensions.mockservices` ?

Reply:
I prefer `serviceextensions.mockservices` as one can misuse by adding classes to `service` level later. WDYT ?